src/app/scripts/cac/pages/cac-pages-directions.js

```diff
@@ -23,17 +23,55 @@ CAC.Pages.Directions = (function ($, _, DirectionsList, Itinerary, Settings, Use
     }
 
     Directions.prototype.initialize = function () {
-        var params = Utils.getUrlParams();
-        if (!_.has(params, 'itineraryIndex')) {
-            // TODO: show this error in the UI
+
+        // Note: date/time does not get passed and so always defaults to departing now
+
+        var rawParams = location.search.slice(1).split('&');
+        var itineraryIndex = -1; // initialize to flag value for missing index parameter
+
+        // process parameters for consumption by OpenTripPlanner:
+        // strip out itineraryIndex, and convert singe waypoint parameter into
+        // multiple intermediatePlaces parameters.
+        var otpParams = _.map(rawParams, function(paramStr) {
```

Comment:
I don't feel super about all the parameter-munging happening here; open to any suggestions on how to make this cleaner.